src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -288,7 +290,7 @@
      (fn []
        ~@body)))
 
-(defn with-query-results-cursor*
+(defn with-query-results-cursor
```

Comment:
Totally fine with it being in a different PR. My only real objection is that 'with-blah' usually is a macro, so it's a little surprising to see it as a function which takes a callback.